integration_test/test_worker/test_worker.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import time
+
+from omotes_sdk.internal.worker.worker import initialize_worker, UpdateProgressHandler
+
+logger = logging.getLogger("test_worker")
+
+
+def test_worker_task(
```

Comment:
Should we (in the future) also test exception thrown?

Reply:
Yep definitely! Once we have a proper framework for integration tests, having a worker that causes certain error conditions is very worthwhile having. Once we implement that the workflow definitions are read from file rather than hardcoded, we don't have to add the test workers in the workflow definitions and we can create test workers for all kinds of scenarios.